pom.xml

```diff
@@ -1194,11 +1195,23 @@
             <dependency>
                 <groupId>org.testcontainers</groupId>
                 <artifactId>testcontainers-bom</artifactId>
-                <version>1.12.4</version>
+                <version>${dep.testcontainers.version}</version>
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
 
+            <dependency>
+                <groupId>org.testcontainers</groupId>
+                <artifactId>testcontainers</artifactId>
+                <version>${dep.testcontainers.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>org.slf4j</groupId>
+                        <artifactId>slf4j-api</artifactId>
```

Comment:
I assume this is for a version conflict? If so, we should either update our own dependency on it, or suppress the version check. Excluding the dependency is wrong as it’s actually needed.

Reply:
@electrum https://github.com/airlift/airbase/pull/214